src/main/java/ai/deepcode/javaclient/core/AnalysisDataBase.java

```diff
@@ -322,7 +320,7 @@ private boolean isNotSucceed(@NotNull Object project, EmptyResponse response, St
     pdUtils.progressSetText(progress, PREPARE_FILES_TEXT);
     dcLogger.logInfo(PREPARE_FILES_TEXT);
     pdUtils.progressCheckCanceled(progress);
-    Map<String, String> mapPath2Hash = new HashMap<>();
+    FileHashRequest hashRequest = new FileHashRequest();
```

Comment:
There's `var` keyword in >Java10, right? It's more readable with it and more aligned with C# implementation when it comes to having code comprehensible in cross-functional team.

Reply:
we build on java 8 to support Idea < 2021.1 and (potentially) other IDE (like Eclipse)

Reply:
@ArtsiomCh thanks! https://github.com/snyk/code-sdk-java/blob/75041d185055841a7ce25ca569af98143a4d488a/build.gradle#L37 this has confused me since I've looked at the wrong setting :) Why do we compile differently between tests and target systems? 👀